toolkit/scss/forms/_questions.scss

```diff
@@ -12,6 +12,11 @@
   .hint {
     margin-bottom: 0;
   }
+
+  // This preserves the vertical margins set on the .question-headings
+  legend {
```

Comment:
Can we check this with @ralph-hawkins? This will add 10px to the spacing between legend and first field so we should make sure we want this.

This doesn't mean I'm against this exact change, it's just worth asking if we need to also adjust the padding and line-height so the behaviour of the CSS is correct but the visuals stay the same.